internal/service/implement/transaction_service.go

```diff
@@ -204,7 +225,53 @@ func (service *TransactionService) InternalTransfer(ctx *gin.Context, transferRe
 		return nil, err
 	}
 
-	// notify, response history
+	// send notification to ws
+	notificationForSourceCustomerResp := &model.NotificationResponse{
+		DeviceId:      int(sourceCustomer.Id),
+		Name:          sourceCustomer.Name,
+		Amount:        int(existsTransaction.Amount),
+		TransactionId: existsTransaction.Id,
+		Type:          "outgoing_transfer",
+		CreatedAt:     existsTransaction.CreatedAt,
+	}
+
+	notificationForTargetCustomerResp := &model.NotificationResponse{
+		DeviceId:      int(targetCustomer.Id),
+		Name:          targetCustomer.Name,
+		Amount:        int(existsTransaction.Amount),
+		TransactionId: existsTransaction.Id,
+		Type:          "incoming_transfer",
+		CreatedAt:     existsTransaction.CreatedAt,
+	}
+
+	// // notify, response history
+	sourceContent, _ := json.Marshal(notificationForSourceCustomerResp)
+	notificationForSourceCustomer := &entity.Notification{
+		Type:    notificationForSourceCustomerResp.Type,
+		Content: string(sourceContent),
+		IsSeen:  false,
+		UserID:  sourceCustomer.Id,
+	}
+	err = service.notificationRepository.CreateCommand(ctx, notificationForSourceCustomer)
+	if err != nil {
+		return nil, err
+	}
+
+	targetContent, _ := json.Marshal(notificationForSourceCustomerResp)
+	notificationForTargetCustomer := &entity.Notification{
+		Type:    notificationForTargetCustomerResp.Type,
+		Content: string(targetContent),
+		IsSeen:  false,
+		UserID:  targetCustomer.Id,
+	}
+	err = service.notificationRepository.CreateCommand(ctx, notificationForTargetCustomer)
```

Comment:
nên save trong notifcation client

Reply:
already move CreateCommand to notificationClient